pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -1109,8 +1138,15 @@ func aceToRules(ctx *zedrouterContext, aclArgs types.AppNetworkACLArgs,
 		}
 	}
 	if foundDrop {
-		outActions = append(outActions, []string{"-j", "DROP"}...)
-		inActions = append(inActions, []string{"-j", "DROP"}...)
+		if aclArgs.NIType == types.NetworkInstanceTypeSwitch {
+			inActions = append(inActions, []string{"-j", "DROP"}...)
+			outActions = append(outActions, []string{"-j", "DROP"}...)
+		} else {
+			inActions = append(inActions, []string{"-j", "LOG", "--log-prefix",
+				"FORWARD:TO:", "--log-level", "3"}...)
+			outActions = append(outActions, []string{"-j", "LOG", "--log-prefix",
+				"FORWARD:FROM:", "--log-level", "3"}...)
```

Comment:
Would these logs be useful even in the switch case?

Reply:
Actually I'm not sure about all these logs. I have a task assigned to consider whether to keep or get rid of the iptables logs in favor of flow-logs. But since we are unable to collect flowlogs for switch networks (there are no conntracks), we should probably add/keep logging for switch NIs.

Reply:
Done. Added logs just before explicit drop rules for switch networks.